src/common/models/measure/measure.ts

```diff
@@ -69,19 +69,24 @@ export class Measure extends BaseImmutable<MeasureValue, MeasureJS> {
     return deduplicateSort(references);
   }
 
-  /**
-   * Look for all instances of countDistinct($blah) and return the blahs
-   * @param ex
-   * @returns {string[]}
-   */
-  static getCountDistinctReferences(ex: Expression): string[] {
-    let references: string[] = [];
+  static isCountDistinctReferences(ex: Expression): boolean {
+    let isCountDistinct = false;
     ex.forEach((ex: Expression) => {
```

Comment:
use `ex.some`

Reply:
are you sure?
https://github.com/allegro/turnilo/blob/7e75477dadd97192d4614eb51a383983ad05b31a/src/common/models/measure/measure.ts#L195

Reply:
Maybe we should reimplement this as some util? `let` + `forEach` always makes me worried about future.